src/WebJobs.Script.WebHost/Models/HostAssignmentContext.cs

```diff
@@ -21,6 +21,8 @@ public class HostAssignmentContext
         [JsonProperty("lastModifiedTime")]
         public DateTime LastModifiedTime { get; set; }
 
+        public MSISpecializationPayload MSISpecializationPayload { get; set; }
```

Comment:
This doesn't need JsonProperty attribute?

Reply:
Fixed